index.html

```diff
@@ -333,9 +333,6 @@
           The <a>PaymentRequest</a> constructor MUST act as follows:
         </p>
         <ol data-link-for="PaymentDetails" class="algorithm">
-          <li>If a <code>paymentRequestId</code> was not provided during
```

Comment:
Here, I think we want this instead:

```HTML
          <li>Establish the request's id:
            <ol>
              <li>Let <var>requestId</var> be the empty string.
              </li>
              <li>If <var>details.paymentRequestId</var> is present, set <var>
                requestId</var> to the value of
                <var>details.paymentRequestId</var>. Otherwise, generate a
                string to uniquely identify this payment request and set
                <var>requestId</var> to that value. It is RECOMMENDED that the
                generated string be a <abbr title=
                "Universally Unique Identifier">UUID</abbr> as per
                [[!RFC4122]].
              </li>
            </ol>
          </li>
```

Then, after the object is actually constructed:

```HTML
          <li>Set <var>request</var>.<a>paymentRequestId</a> to
          <var>requestId</var>.
```